email.js

```diff
@@ -12,7 +12,7 @@ const transporter = nodemailer.createTransport({
 });
 
 async function sendConfirmationEmail(email, eventID, registrantID) {
-  const link = `http://localhost:3000/confirmEmail/${eventID}/${registrantID}`;
+  const link = config.email.link.emailConfirmation;
   let info = await transporter.sendMail({
     from: '"LICCB Mailer 🚣" <liccb.mailer@gmail.com>', // sender address
```

Comment:
is this value hard-coded separately in multiple places, like here on 17, and below on line 28?  Anywhere else?  This needs to be parameterized in a config file

Reply:
@cjlandolfi any info on this?